app/models/concerns/user_otp_methods.rb

```diff
@@ -15,7 +15,7 @@ def create_direct_otp
   end
 
   def generate_totp_secret
-    ROTP::Base32.random_base32(16)
+    ROTP::Base32.random(20)
```

Comment:
what if we left a comment here describing the # of bits?#5604 
```suggestion
    ROTP::Base32.random(20) # 160-bit secret, per RFC 4226
```